### DISCUSSION TITLE
有考虑出物品记录和睡眠记录功能吗

### DISCUSSION THREAD

**Discussion by Cheerssherly** [WatsonYao/nian_app#87] [Q&A] [answered]
特别特别需要(⁠•⁠‿⁠•⁠)

**Comment by WatsonYao** [comment, marked as answer]
因为手环类的睡眠应用已经非常完善了，暂不会考虑。物品类的可能会。

**Reply by Cheerssherly** [reply]
太好了，确定感觉现在市面上还没有特别好记录物品的软件，感谢(⁠☆⁠▽⁠☆⁠)

**Comment by WatsonYao** [comment]
方便展开一下所说的物品记录功能了，平常是要解决什么问题，举一些常用的例子。

**Reply by Cheerssherly** [reply]
主要记录比如衣服、药品、食品、电子产品之类的东西。因为东西太多了，想要统一了解自己有的东西，也算是种复盘。大概需要的功能是记录物品数量、标签分类、购买日期、购买价格、拆封使用日期、使用频率、保质期、保修期、评价、存放位置，要是能额外给电子产品记录下充电频率和使用时间就更好了。

**Reply by Cheerssherly** [reply]
主要想要统计的功能，最好可以加上照片，之前用excel记录最大缺点就是不好加照片TT
